### ISSUE TITLE
Download Chapters From Tapas

### ISSUE THREAD

**Issue by Souravi785** [fmd-project-team/FMD#469]
How do I download the locked chapters from Tapas?

**Comment by SDXC** [comment]
Do you have an account and can you read the locked chapters on the website?

**Comment by Souravi785** [comment]
Yes, I have account on Tapas. But there is no option to add Tapas in FMD.

**Comment by Macachan** [comment]
- Open Tapas in Firefox
- Log in
- Click on a bought chapter and go to "Tools" ->  "Developer" -> "Network"
- Click on "HTML" under "Network" tab
- On the right side of "HTML", you can see the tab "Headers"
- Go to "Request Headers" and spot "User-Agent" and "Cookie" 
- Copy the values from these two to a notepad and use them intact on FMD as shown in the image:

![FMD Tapas](https://user-images.githubusercontent.com/56768361/67148830-d0a0b300-f2ac-11e9-946e-c8381f8babf9.png)

The only values you mess with are the ones shown in the red rectangles. The other ones under "CloudflareBypass" change automatically after you're done loading the url of the comic/manga you are interested in.

Don't forget to click on "Apply"!

To get the manga info, simply copy-paste the url of the series from your browser to FMD, keeping in mind it must be in a "https" format.

For further understanding please refer here: [https://github.com/fmd-project-team/FMD/wiki](url), under "Cloudflare Workarounds".
